AUTHORS

```diff
@@ -0,0 +1,20 @@
+Ahmed Ibrahim <me@ahmedibrahim.dev>
+blaumeise20 <62756994+blaumeise20@users.noreply.github.com>
+CharlesChen0823 <yongchen0823@gmail.com>
+Dibash Thapa <dibashthapa55@gmail.com>
+Echo <echo-the-coder@tuta.io>
+Eric Walker <emwalker@users.noreply.github.com>
+Eric Walker <eric.walker@gmail.com>
```

Comment:
Can we drop the one with "eric.walker@...", since both of these people are me?

Reply:
The list is generated from the commits in the git repo. The problem with removing is that it will be added again with a new "make authors". Unless you really have an issue with it, i would like to keep it